<ui version="4.0" >
 <class>Kleo::NewCertificateUi::ChooseProtocolPage</class>
 <widget class="QWizardPage" name="Kleo::NewCertificateUi::ChooseProtocolPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>294</height>
   </rect>
  </property>
  <property name="title" >
   <string comment="@title" >Choose Certificate Format</string>
  </property>
  <property name="subTitle" >
   <string>Please choose which type of certificate you want to create. </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="sizeConstraint" >
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QCommandLinkButton" name="pgpCLB" >
     <property name="text" >
      <string>Create a personal OpenPGP key pair</string>
     </property>
     <property name="description" >
      <string>OpenPGP key pairs are created locally, and certified by your friends and acquaintances. There is no central certification authority; instead, every individual creates a personal Web Of Trust by certifying other user's key pairs with his own certificate.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="x509CLB" >
     <property name="text" >
      <string>Create a personal X.509 key pair and certification request </string>
     </property>
     <property name="description" >
      <string>X.509 key pairs are created locally, but certified centrally by a certification authority (CA). CAs can certify other CAs, creating a central, hierarchical chain of trust.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pgpCLB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ChooseProtocolPage</receiver>
   <slot>slotPgpClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>642</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel" >
     <x>692</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x509CLB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ChooseProtocolPage</receiver>
   <slot>slotX509Clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>592</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel" >
     <x>688</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotPgpClicked()</slot>
  <slot>slotX509Clicked()</slot>
 </slots>
</ui>
